test.sh

```diff
@@ -5,9 +5,9 @@ TEST_USER='test@sentry.io'
 TEST_PASS='test123TEST'
 COOKIE_FILE=$(mktemp)
 declare -a TEST_STRINGS=(
-    '"isAuthenticated":true,'
-    '"username":"test@sentry.io",'
-    '"isSuperuser":true,'
+    '"isAuthenticated":true'
+    '"username":"test@sentry.io"'
```

Comment:
Do we maybe wanna put a less potentially real email? I'm not sure where this goes, but also not sure if there's value in there being a real domain either.

Reply:
This is used in the test script and flow. Happy to change it to something more random in a follow-up PR.